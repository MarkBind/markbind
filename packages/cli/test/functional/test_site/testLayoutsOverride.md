<frontmatter>
title: Test Layouts
head: overwriteLayoutHead.md
layout: default
</frontmatter>

**Uses a site.json layout, overriding frontmatter**
