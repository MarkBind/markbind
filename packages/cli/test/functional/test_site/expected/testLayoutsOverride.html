<!DOCTYPE html>
<html>

<head>

  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="MarkBind 2.16.0">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Hello World</title>
  <link rel="stylesheet" href="markbind/css/bootstrap.min.css">
  <link rel="stylesheet" href="markbind/css/bootstrap-vue.min.css">
  <link rel="stylesheet" href="markbind/fontawesome/css/all.min.css">
  <link rel="stylesheet" href="markbind/glyphicons/css/bootstrap-glyphicons.min.css">
  <link rel="stylesheet" href="markbind/css/octicons.css">
  <link rel="stylesheet" href="markbind/css/github.min.css">
  <link rel="stylesheet" href="markbind/css/markbind.min.css">
  <script src="markbind/js/polyfill.min.js"></script>
  <script src="markbind/js/vue.min.js"></script>
  <script src="markbind/js/markbind.min.js"></script>
  <script src="markbind/js/jquery.min.js"></script>
  <link rel="stylesheet" href="/test_site/plugins/testMarkbindPlugin/testMarkbindPluginStylesheet.css">
  <link rel="stylesheet" href="/test_site/plugins/markbind-plugin-anchors/markbind-plugin-anchors.css">
  <link rel="stylesheet" href="markbind/layouts/testLayout/styles.css">
  <script src="/test_site/headFiles/overwriteLayoutScript.js"></script>
  <link rel="icon" href="/test_site/favicon.png">
</head>

<body>
  <div id="app">
    <div id="flex-body">
      <nav id="site-nav" class="navbar navbar-light bg-transparent">
        <div class="border-right-grey nav-inner position-sticky slim-scroll">
          <ul class="site-nav-list site-nav-list-root">
            <li class="site-nav-custom-list-item site-nav-list-item-0">[Layout Nav]</li>
          </ul>
        </div>
      </nav>
      <div id="content-wrapper">
        <p><strong>Uses a site.json layout, overriding front matter</strong></p>
        <p><i class="fa fa-arrow-circle-up fa-lg" id="scroll-top-button" onclick="handleScrollTop()" aria-hidden="true"></i></p>
      </div>


    </div>
    <footer>
      <div class="text-center">
        Layout footer
      </div>
    </footer>
  </div>
</body>
<script src="markbind/js/bootstrap-utility.min.js"></script>
<script>
  const baseUrl = '/test_site'
  MarkBind.setupWithSearch()
</script>
<script src="/test_site/plugins/testMarkbindPlugin/testMarkbindPluginScript.js"></script>
<script>
  alert("Inline plugin script loaded!")
</script>

<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=TRACKING-ID"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
    dataLayer.push(arguments);
  }
  gtag('js', new Date());

  gtag('config', 'TRACKING-ID');
</script>
<script src="markbind/layouts/testLayout/scripts.js"></script>
<script>
  if (MarkBind.AfterSetup) MarkBind.AfterSetup()
</script>

</html>